tools/c7n_gcp/pyproject.toml

```diff
@@ -18,6 +18,9 @@ classifiers = [
 python = "^3.7"
 retrying = "^1.3.3"
 google-api-python-client = "^2.0"
+# pin these for now
```

Comment:
these were the offending lines that needed to be pinned to get ci past the first initial failures